Assets/Scripts/ManagerScripts/CountdownManager.cs

```diff
@@ -9,38 +9,55 @@ public class CountdownManager: MonoBehaviour
         public TMP_Text countdownText;
         public GameObject countdownUI;
         public AudioSource countdownSound;
-        private bool _shouldCountTime;
+        public bool shouldCountTime;
         private bool _soundPlayed;
+        private bool _gameStart;
+        
 
         private void Start()
         {
-            _shouldCountTime = SceneManager.GetActiveScene().name != "MainMenuScene";
+            shouldCountTime = SceneManager.GetActiveScene().name != "MainMenuScene";
             countdown = 5f;
+            _gameStart = true;
         }
         
         private void Update()
         {
-            if ( _shouldCountTime && !_soundPlayed)
+            if ( shouldCountTime && !_soundPlayed)
```

Comment:
Side note: Doesn't this if block get triggered every frame after the countdown ends? 

- Let's say we're in level one scene, so shouldCountTime is always true (since it never changes after it gets set)
- _soundPlayed after the countdown is set to false, which means this if-block gets triggered after the countdown, and then _soundPlayed gets set to true. 
- Since countdown <= 0, the else-block will get triggered which does a bunch of stuff and sets _soundPlayer to false
- ...which means the if-block and else-block will get triggered every frame

Lol I think the countdown sound is played and stopped within the same frame every frame, that's why we don't hear it. Should probably fix this. Im guessing this has some negative performance effect 🤷

Reply:
Added a log statement in the if-block. Confirmed: 
![image](https://user-images.githubusercontent.com/45300621/204446099-9cdc9a72-cda0-4e0e-9d82-1e107d27d409.png)

Reply:
I think this should work? 
![image](https://user-images.githubusercontent.com/45300621/204446613-c26c5690-e7c9-40ea-bb60-398268dbc148.png)